.github/workflows/e2e-test.yml

```diff
@@ -11,11 +11,20 @@ on:
         description: 'The hash value of the commit'
         required: false
         default: ''
+      run-eol-python-version:
```

Comment:
Would it make sense to include another variable here for us to specify which python version we want to run against? E.g. 3.12

Reply:
Yeah, I think that's a good idea! It will be really nice to be able to run the tests against all supported python versions easily.